js/build.js

```diff
@@ -0,0 +1,344 @@
+Fliplet.Widget.instance({
+  name: 'file-list',
+  displayName: 'File list',
+  icon: 'fa-exchange',
+  render: {
+    dependencies: [],
+    template: [
+      '<div data-view="content"></div>',
+    ].join(''),
+    ready: async function () {
+      var thisy = this;
```

Comment:
What's this for? You should be able to access `this` when using arrow functions.

**Furthermore, using `var` is deprecated. Use `const` or `let` throughout the entire file.**

Reply:
That was the issue I was talking about, when using the fat arrow function, **this** is not available, it is undefined for some reason but still, not always.